structuralalignment/superposition/theseus.py

```diff
@@ -1 +1,114 @@
-#here is the theseus code
+import atomium
+import subprocess
+
+# usage: Theseus(["3dk3"])
+class Theseus:
+    def __init__(self, pdbs):
+        self.pdbs = pdbs
+        self.fetched_pdbs = []
+        for pdb in pdbs:
+            self.fetched_pdbs.append(atomium.fetch(pdb))
+        self.align()
```

Comment:
Since this will be a factory, we can't call `.align` at initialization. It should be a separate step:

```
aligner = Theseus()
aligner.align(pdb1, pdb2)
```